README.md

```diff
@@ -7,18 +7,66 @@
+permissions. This only needs to be run once per project, per AWS
+account. This user can also be used to run the Molecule tests on your
+local machine.
+
+Before the build user can be created, you will need a profile in your
+AWS credentials file that allows you to read and write your remote
+Terraform state.  (You almost certainly do not want to use local
+Terraform state for this long-lived build user.)  If the build user is
+to be created in the CISA COOL environment, for example, then you will
+need the `cool-terraform-backend` profile.
+
+The easiest way to set up the Terraform remote state profile is to
+make use of our
+[`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync)
+utility. Follow the usage instructions in that repository before
+continuing with the next steps, and note that you will need to know
+where your team stores their remote profile data in order to use
+[`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync).
+
+To create the build user, follow these instructions:
+
+```console
+cd terraform
+terraform init --upgrade=true
+terraform apply
+```
+
+Once the user is created you will need to update the [repository's
+secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
+with the new encrypted environment variables. This should be done
+using the
+[`terraform-to-secrets`](https://github.com/cisagov/development-guide/tree/develop/project_setup#terraform-iam-credentials-to-github-secrets-)
+tool available in the [development
+guide](https://github.com/cisagov/development-guide). Instructions for
+how to use this tool can be found in the ["Terraform IAM Credentials
+to GitHub Secrets"
+section](https://github.com/cisagov/development-guide/tree/develop/project_setup#terraform-iam-credentials-to-github-secrets-).
+of the Project Setup README.
+
+If you have appropriate permissions for the repository you can view
+existing secrets on the [appropriate
+page](https://github.com/cisagov/ansible-role-nessus/settings/secrets)
+in the repository's settings.
+
 ## Requirements ##
 
 None.
 
 ## Role Variables ##
 
-- `package_bucket` - The name of the AWS S3 bucket containing the Nessus
-  Debian package file.  Default value: `"cisa-cool-third-party-production"`
-- `package_file` - The name of the Nessus Debian package file.
-  Default value: `"Nessus-{{ version }}-debian6_amd64.deb"`
-- `version` - The version number of the Nessus Debian package file.
-  Default value: `"8.7.1"`
+| Variable | Description | Default | Required |
+|----------|-------------|---------|----------|
+| package_bucket | The name of the AWS S3 bucket containing the Nessus Debian package file. | `cisa-cool-third-party-production` | No |
+| package_file | The name of the Nessus Debian package file. | `Nessus-{{ version }}-debian6_amd64.deb` | No |
+| version | The version number of the Nessus Debian package file. | `8.7.1` | No |
```

Comment:
I was speaking with @mcdonnnj and he mentioned that he thought you would be updating us to the newest version of Nessus 8.x (currently `8.15.4`).  I just wanted to confirm if you planned to do that in a separate PR or if you were going to include it in this one.